client/my-sites/plugins/plugin-icon/plugin-icon.tsx

```diff
@@ -1,10 +1,15 @@
 import { Gridicon } from '@automattic/components';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 
 import './style.scss';
 
-const PluginIcon = ( { className, image, isPlaceholder } ) => {
+declare interface PluginIconProps {
```

Comment:
Curious why the declare keyword is used here?

Reply:
Agree, looks like that's identical to just using `interface` and looks like that's the standard in calypso

Reply:
Thanks! Looks like it is the standard in Calypso, so have removed. 👍